data/resources/snap_is_being_refreshed_gtk4.cmb

```diff
@@ -1,50 +0,0 @@
-<?xml version='1.0' encoding='UTF-8' standalone='no'?>
```

Comment:
Why do you delete the Cambalache base file? I mean: the .ui file is generated from this one. I think that it's better to use a graphical interface to define the dialog windows...

Reply:
Can Cambalache not open .ui files directly? I haven't used it before and because I'd changed to using a template the .cmb file no-longer matches (AFAIK). Can Cambalache support templates?

Reply:
I've tried running with Cambalache (from the git branch) and it doesn't load the .cmb file that's already in this project. The .ui does load in Glade though - I feel like Cambalache isn't sufficiently stable or easy to install/depend on here.

Reply:
I used the flatpak version (there is no snap version... we must fix that).